src/styles/reset.scss

```diff
@@ -0,0 +1,11 @@
+body,
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+body {
+  font-family: "Arial", sans-serif;
+  font-size: 30px;
+}
```

Comment:
let's move font properties to the `:root` or `html` tag

Reply:
Good point. Move this to normalize and set props to root tag, as your mate advised.